picard/tagger.py

```diff
@@ -244,6 +245,7 @@ def __init__(self, picard_args, localedir, autoupdate, pipe_handler=None):
         self.__class__.__instance = self
         setup_config(self, picard_args.config_file)
         config = get_config()
+        init_settings_groups()
```

Comment:
This method should be called from `setup_config()`, it has to be executed before any access to config, and after options were defined.